common/app/experiments/Experiments.scala

```diff
@@ -55,3 +56,12 @@ object MastheadWithHighlights
       sellByDate = LocalDate.of(2024, 9, 30),
       participationGroup = Perc0C,
     )
+
+object DeeplyReadRightColumn
```

Comment:
We don't need to create an Experiment for a client-side test. This file is used to create server-side tests. Only steps 1-3 are needed from the [guide](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/docs/development/ab-testing-in-dcr.md#quick-start).